setup.py

```diff
@@ -5,7 +5,7 @@
       version='1.0a1',
       install_requires=[
             'sbol3',
-            'rdflib>=5.0.0',
+            'rdflib<6.0.0',
```

Comment:
Can you change this to indicate "between" rather than only less than or greater than?